packages/types/src/event.ts

```diff
@@ -22,10 +22,11 @@ export const EventSchema = schemas.EventSchema.extend({
 
 export const EventTypeSchema = schemas.EventTypeSchema
 export const EventStatusSchema = schemas.EventStatusSchema
+export const EventStatusWriteSchema = schemas.EventStatusSchema.exclude(["DELETED"])
+export type EventStatusWrite = z.infer<typeof EventStatusWriteSchema>
```

Comment:
What is this used for?

Reply:
Creating drop downs for status in edit and create event form

Reply:
Hmm okay, although I would argue if its not used elsewhere, just declare that in the dashboard code